generated/nirfmxinstr/nirfmxinstr.proto

```diff
@@ -211,6 +211,106 @@ enum NiRFmxInstrAttribute {
   NIRFMXINSTR_ATTRIBUTE_THERMAL_CORRECTION_TEMPERATURE_RESOLUTION = 120;
 }
 
+enum Boolean {
```

Comment:
@harsha-bhushan this is the second change adding enums to grpc-device for RFmxInstr. You can see all of the enums and attributes now in the proto file.

Does this look OK? Is it what you expected? Any questions/comments/concerns?